ED/protocol.html

```diff
@@ -1140,6 +1160,12 @@ <h3 property="schema:name">Web Access Control</h3>
                   <p>[<a href="https://github.com/solid/specification/issues/171" rel="cito:citesAsSourceDocument">Source</a>] [<a href="https://github.com/solid/specification/issues/170" rel="cito:citesAsSourceDocument">Source</a>] <a href="https://github.com/solid/specification/issues/181" rel="cito:citesAsSourceDocument">Source</a>] <a href="https://gitter.im/solid/specification?at=60101295d8bdab47395e6775" rel="cito:citesAsSourceDocument">Source</a>]</p>
                 </div>
               </section>
+              <section id="access-control-policy" inlist="" rel="schema:hasPart" resource="#access-control-policy">
+                <h3 property="schema:name">Access Control Policy</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>Access Control Policy (<abbr title="Access Control Policy">ACP</abbr>) is an authorization language that uses a set of rules to describe, control, and grant access to Web resources. Based on a decentralized Access Control List (ACL) model, the Solid server uses ACP to manage associations to a resource and applies authorization rules on requested operations. Authorizations are described using the ACL ontology to express and determine access privileges of a requested resource.</p>
+                </div>
```

Comment:
I wonder if we want to describe WAC and ACP together somehow; because there is some overlap in their descriptions, and some differences For instance, WAC describes itself as "decentralized cross-domain access control", and ACP as "Based on a decentralized Access Control List (ACL) model". So is ACP then cross-domain or not? Is this a difference? Etc.